config/rbac/service_account.yaml

```diff
@@ -53,4 +53,3 @@ subjects:
 - kind: ServiceAccount
   name: designate
   namespace: openstack
```

Comment:
do we know which is the right one to delete?

Reply:
may be @dkehn can help us here.